layers/+completion/ivy/config.el

```diff
@@ -47,3 +47,6 @@ than this amount.")
         for desc = (nth 2 j)
         collect `(,key (lambda (x) (spacemacs//counsel-with-git-grep (quote ,func) x)) ,desc))
   "Default ivy actions to be used with git-grep output.")
+
+(defvar spacemacs--ivy-rich nil
```

Comment:
This layer variable should be something like `ivy-enable-ivy-rich`. `spacemacs--*` are, I think, private/internal.

Reply:
Okay I doubted that, I'll update it. Thanks!